webpack.config.js

```diff
@@ -513,15 +513,28 @@ async function initializeModuleFederation() {
 
             // Other containers will be forced to use the exact versions of shared modules that the web app provides.
             makeSingletonSharedModules([
+                'history',
                 'react',
+                'react-beautiful-dnd',
```

Comment:
why did we had to add this one?

Reply:
It depends on a singleton embedded somewhere in the global context that contains all the current draggables. If we have 2 instances running the library complains a lot, so it's better to merge them.